lib/Backend/GlobOptBailOut.cpp

```diff
@@ -916,7 +916,7 @@ GlobOpt::MayNeedBailOnImplicitCall(IR::Opnd * opnd, Value *val, bool callsToPrim
     case IR::OpndKindReg:
         // Only need implicit call if the operation will call ToPrimitive and we haven't prove
         // that it is already a primitive
-        return callsToPrimitive &&
```

Comment:
was the problem here that some opcodes weren't marked with OpCallsValueOf attribute and this method would basically say that there is no need of a bailout on implicit calls for this opnd?